Asterix/main_THD.py

```diff
@@ -143,6 +143,10 @@ def runthd2(parameter_file_path,
                           save_for_bench=onbench,
                           realtestbed_dir=labview_dir)
 
+    ### Write configfile to Labview-style matrix directory
+    config.filename = os.path.join(labview_dir, "Simulation_parameters.ini")
```

Comment:
Axel had dvp a tool to save all params to fits header also `from_param_to_header` in utils/save_and_read.py module. 
`header = from_param_to_header(config)`
This could be also interesting so that each configuration is attached to the matrix file itself

Reply:
References in #152 - I would like to merge this here asap because we really need it when working on the testbed.

Reply:
ok

Reply:
ok